build.gradle

```diff
@@ -10,26 +10,26 @@ buildscript {
 plugins {
     id 'java'
     id "com.github.spotbugs" version "4.2.4" apply false
+    id "com.adarshr.test-logger" version "2.1.1" apply false
 }
 
 allprojects {
     apply plugin: "com.github.spotbugs"
 
-    group 'com.velocitypowered'
+    group 'gg.solarmc'
     version '1.1.5-SNAPSHOT'
 
     ext {
         // dependency versions
         textVersion = '3.0.4'
         adventureVersion = '4.7.0'
-        adventurePlatformVersion = '4.0.0-SNAPSHOT'
         junitVersion = '5.7.0'
         slf4jVersion = '1.7.30'
-        log4jVersion = '2.13.3'
+        log4jVersion = '2.14.0'
         nettyVersion = '4.1.58.Final'
         guavaVersion = '25.1-jre'
         checkerFrameworkVersion = '3.6.1'
-        configurateVersion = '3.7.1'
```

Comment:
removing configurate?

Reply:
I didn't think we would use it considering we have dazzleconf, as well as your config system if you like. However if we need it we can always add it back.

Configurate and tomlj were the velocity dependencies I saw which velocity itself didn't use at all.

Reply:
I'd rather not use Gears here, as it's pretty NOT lightweight and depends on Beetle, and you know i don't like using dependencies where they aren't absolutely necessary. I'd still consider keeping configurate since i'm pretty sure quite a few existing velocity plugins rely on it being packaged with velocity, but then again we aren't going to be using many other plugins besides purpose built ones

Reply:
We can keep both configurate and tomlj too if needed. They just won't be declared in the velocity build.gradle files, because velocity does not depend on them. However, we can add them at runtime to the modulepath. After all, shading dependencies in the server was thrown out the window in the PR to modularize SolarPaper, and likewise will it be discarded when we modularize Velocity.